include/SugarFields/Fields/Multienum/SugarFieldMultienum.php

```diff
@@ -58,7 +58,15 @@ function getSearchViewSmarty($parentFieldArray, $vardef, $displayParams, $tabind
     	}
     }
 
-    function displayFromFunc( $displayType, $parentFieldArray, $vardef, $displayParams, $tabindex ) {
+    /**
+     * @param $displayType
+     * @param $parentFieldArray
+     * @param $vardef
+     * @param $displayParams
+     * @param int $tabindex
+     * @return string
+     */
+    function displayFromFunc( $displayType, $parentFieldArray, $vardef, $displayParams, $tabindex = 0) {
```

Comment:
Note: please next, when refactoring a function definition, add the visibility of method like `private`, `public` etc.